shell/assets/translations/en-us.yaml

```diff
@@ -5728,6 +5728,9 @@ validation:
   setting:
     serverUrl:
       https: server-url must be https.
```

Comment:
I think it would be good to update this one to match the new error messages: `Server URL must be https.` Otherwise PR looks good